Scripts/Playthrough.cs

```diff
@@ -374,6 +341,235 @@ private bool IsConnected()
             return _connectionState == ConnectionState.Connected;
         }
 
+        private bool HasBeenDisconnected()
+        {
+            return _connectionState == ConnectionState.Disconnected;
+        }
+
+        private async Task CreateClientAndJoinRoom()
```

Comment:
since this method doesn't return anything, should this be `void`?
```suggestion
        private async void CreateClientAndJoinRoom()
```